channels/kh.m3u

```diff
@@ -1,22 +1,8 @@
 #EXTM3U
-#EXTINF:-1 tvg-id="BayonTV.kh" tvg-name="Bayon TV" tvg-country="KH" tvg-language="Khmer" tvg-logo="https://i.imgur.com/2jbulL2.jpg" group-title="News",Bayon TV (360p)
-http://203.176.130.123:8989/live/bayontv1_480k.stream/playlist.m3u8
-#EXTINF:-1 tvg-id="BayonTV.kh" tvg-name="Bayon TV" tvg-country="KH" tvg-language="Khmer" tvg-logo="https://i.imgur.com/1TgC7mx.jpg" group-title="News",Bayon TV (360p)
-http://203.176.130.123:8989/live/bayontv_480k.stream/playlist.m3u8
 #EXTINF:-1 tvg-id="FreshNews.kh" tvg-name="Fresh News" tvg-country="KH" tvg-language="" tvg-logo="" group-title="",Fresh News
 http://167.99.65.12:1935/live/myStream/playlist.m3u8
 #EXTINF:-1 tvg-id="FreshNewsTV.kh" tvg-name="Fresh News TV" tvg-country="KH" tvg-language="Khmer" tvg-logo="https://i.imgur.com/6p0TK7e.png" group-title="",Fresh News TV (720p)
 http://167.99.65.12:1935/live/ngrp:myStream_all/chunklist.m3u8
-#EXTINF:-1 tvg-id="PNN.kh" tvg-name="PNN" tvg-country="KH" tvg-language="Khmer" tvg-logo="https://i.imgur.com/1xZmAvQ.jpg" group-title="",PNN (360p)
-http://203.176.130.123:8989/live/pnn_480k.stream/playlist.m3u8
-#EXTINF:-1 tvg-id="SEATV.kh" tvg-name="SEA TV" tvg-country="KH" tvg-language="" tvg-logo="" group-title="",SEA TV
-http://203.176.130.123:8989/live/seatv_480k.stream/playlist.m3u8
-#EXTINF:-1 tvg-id="TV9.kh" tvg-name="TV9" tvg-country="KH" tvg-language="Khmer" tvg-logo="https://i.imgur.com/KGbDcg1.jpg" group-title="",TV9 (360p)
-http://203.176.130.123:8989/live/ctv9_480k.stream/playlist.m3u8
-#EXTINF:-1 tvg-id="TVK.kh" tvg-name="TVK" tvg-country="KH" tvg-language="Khmer" tvg-logo="https://i.imgur.com/GD4n70u.png" group-title="",TVK
-http://203.176.130.123:8989/live/tvk_480k.stream/chunklist.m3u8
```

Comment:
Not 24/7

Keeping things "playlist-ish" : http://203.176.130.123:8989/live/tvk_480k.stream/playlist.m3u8